### PR TITLE
Added Contributors in README.md

### PR DESCRIPTION
**Pull request by chippyho123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +29,23 @@ Make sure python3 and pip are installed by running `python3 --version` and `pip3
...
+   (https://sourcerer.io/fame/alexmarginean16/alexmarginean16/codesession/images/3)] 
+   (https://sourcerer.io/fame/alexmarginean16/alexmarginean16/codesession/links/3)[![] 
+   (https://sourcerer.io/fame/alexmarginean16/alexmarginean16/codesession/images/4)] 
+   (https://sourcerer.io/fame/alexmarginean16/alexmarginean16/codesession/links/4)[![] 
+   (https://sourcerer.io/fame/alexmarginean16/alexmarginean16/codesession/images/5)] 
+   (https://sourcerer.io/fame/alexmarginean16/alexmarginean16/codesession/links/5)[![] 
+   (https://sourcerer.io/fame/alexmarginean16/alexmarginean16/codesession/images/6)] 
+   (https://sourcerer.io/fame/alexmarginean16/alexmarginean16/codesession/links/6)[![] 
+   (https://sourcerer.io/fame/alexmarginean16/alexmarginean16/codesession/images/7)] 
+   (https://sourcerer.io/fame/alexmarginean16/alexmarginean16/codesession/links/7)
```
Comment (alexmarginean16) [inline README.md]: Hey. This isn't working. The links should be all in one line. I made a mistake to the code inserted into the issue #4. Try removing those lines 33-51 and add this:

    ##Contributors

    [](https://sourcerer.io/fame/alexmarginean16/alexmarginean16/codesession/links/0)[](https://sourcerer.io/fame/alexmarginean16/alexmarginean16/codesession/links/1)[](https://sourcerer.io/fame/alexmarginean16/alexmarginean16/codesession/links/2)[](https://sourcerer.io/fame/alexmarginean16/alexmarginean16/codesession/links/3)[](https://sourcerer.io/fame/alexmarginean16/alexmarginean16/codesession/links/4)[](https://sourcerer.io/fame/alexmarginean16/alexmarginean16/codesession/links/5)[](https://sourcerer.io/fame/alexmarginean16/alexmarginean16/codesession/links/6)[](https://sourcerer.io/fame/alexmarginean16/alexmarginean16/codesession/links/7)

Comment (chippyho123) [reply README.md]: sure thing

Comment (chippyho123) [reply README.md]: Should be good to go now!

Comment (chippyho123) [reply README.md]: Good luck with your project!

**Review by alexmarginean16 — APPROVED** [review]
Looks good to me. Thanks!

**Pull request merged**